tests/library/browsertype-connect.spec.ts

```diff
@@ -549,13 +550,11 @@ for (const kind of ['launchServer', 'run-server'] as const) {
       await disconnectedPromise;
       expect(browser.isConnected()).toBe(false);
 
-      const navMessage = (await navigationPromise).message;
-      expect(navMessage).toContain('Connection closed');
-      expect(navMessage).toContain('Closed by');
-      expect(navMessage).toContain(__filename);
```

Comment:
Why drop this line? Are missing the stack now?

Reply:
Resolved offline